spec/forms/find_school_form_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe FindSchoolForm, type: :model, with_audited: true do
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:search_type).with_message("Please select an option") }
```

Comment:
Do we get useful error messages when these tests fail? I'd normally expect them to have a description

Reply:
This is what you get when these fail:
```
     FindSchoolForm validations is expected to validate that :search_type cannot be empty/falsy, producing a custom validation error on failure
     Failure/Error: it { is_expected.to validate_presence_of(:search_type).with_message("Please select an invalid option") }
     
       Expected FindSchoolForm to validate that :search_type cannot be
       empty/falsy, producing a custom validation error on failure, but this
       could not be proved.
         After setting :search_type to ‹nil›, the matcher expected the
         FindSchoolForm to be invalid and to produce the validation error
         "Please select an invalid option" on :search_type. The record was
         indeed invalid, but it produced these validation errors instead:
     
         * search_type: ["Please select an option"]
     # ./spec/forms/find_school_form_spec.rb:5:in `block (3 levels) in <main>'
```
It looks pretty readable to me.

Reply:
Ah nice, looks good :)